02-express-tutorial/01-http-basics.js

```diff
@@ -0,0 +1,26 @@
+const http = require("http");
+
+const server = http.createServer((req, res) => {
+  // console.log(req.method)
```

Comment:
Try not to commit comments like this, people often forget to remove these lines and makes the repo look ugly